Sming/SmingCore/Network/Http/HttpConnection.cpp

```diff
@@ -89,7 +89,7 @@ DateTime HttpConnection::getLastModifiedDate()
 {
```

Comment:
`Sming/SmingCore/Network/Http/HttpConnection.cpp 100644 → 100755`

Fix the permissions here.